src/Extensions/DefaultRecordsExtension.php

```diff
@@ -176,4 +184,82 @@ public function createHomePage(): void
         $block->write();
         $page->publishRecursive();
     }
+
+    public function createBlockPage(): void
```

Comment:
Appreciate that all these variants have been added, however all that is required is one block on the homepage. Check out the Figma link.

Reply:
@DevKCode this block is still appearing on the 'Blocks' page. It needs to go on the home page.